src/libponyrt/sched/cpu.c

```diff
@@ -314,6 +314,10 @@ void ponyint_cpu_core_pause(uint64_t tsc, uint64_t tsc2, bool yield)
       // If it has been 1 billion cycles, pause 1 ms.
       ts.tv_nsec = 1000000;
     }
+    else
+    {
+      ts.tv_nsec = 100000;
```

Comment:
Could this line get a human-oriented comment above this line, like the preceding lines each have?